plugins/services/src/js/components/DeploymentStatusIndicator.js

```diff
@@ -58,17 +59,17 @@ class DeploymentStatusIndicator extends mixin(StoreMixin) {
       return null;
     }
 
+    const deploymentText =
+      deploymentsCount === 1 ? i18nMark("deployment") : i18nMark("deployments");
```

Comment:
I'd rather `// TODO pluralize`. Some languages 🇷🇺 have interesting rules around quantities :D

Reply:
@brandonc conflicting feedback re: pluralization. Would you mind if I changed these back to L10NTODOs?